tools/wt_verify/wt_verify.py

```diff
@@ -184,6 +185,21 @@ def histogram(field, chkpt, chkpt_name):
             elif "leaf" in metadata["page_type"]:
                 leaf.append(metadata[field])
 
+    # Using pandas to calculate stats.
+    df_internal = pd.DataFrame(internal, columns=[field])
+    df_leaf = pd.DataFrame(leaf, columns=[field])
+
+    stats_internal = df_internal.describe().loc[['mean', '50%', 'min', 'max']].transpose()
```

Comment:
You could create variables for:

```
['mean', '50%', 'min', 'max']
['Mean', 'Median', 'Min', 'Max']
```

Reply:
Updated in [94b7f36](https://github.com/wiredtiger/wiredtiger/pull/10230/commits/94b7f3678b2778502e99a117a45e702975c36107)